### ISSUE TITLE
About the epoch of training

### ISSUE THREAD

**Issue by 33KU** [Huage001/AdaAttN#5]
How many epochs have been trained for the model in this paper?
Thanks~

**Comment by Huage001** [comment]
As configured in train_adaattn.sh, there are 2 epochs with full learning rate and another 3 with learning rate decay.
